cli/config/Config-Clarification.md

```diff
@@ -0,0 +1,24 @@
+This document is about some expert config options.
+
+* [disable_new_netns](#disable_new_netns)
+* [internetworking_model=none](#internetworking_model=none)
+
+# disable_new_netns
+
+This option will not create a standalone network namespace for hypervisor (like qemu) and shim processes. They will
+live in the host network namespace.
+
+- This option may have some potential impacts to your host. It should only be used by people who know what they're doing.
+
+- `disable_new_netns` conflicts with `enable_netmon`
+
+- `disable_new_netns` conflicts with `internetworking_model=bridged` and `internetworking_model=macvtap`. It works only
+ with `internetworking_model=none`. This model is used when you want better performance.
```

Comment:
> This model is used when you want better performance.

This description is not good, it seems to indicate that our default network model has poor performance. I think it's better to say something like "This model is used when you want to do more custom works on networks".

I'll leave this to docs team